vms/vms/settings.py

```diff
@@ -61,8 +61,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'myuser',
-        'PASSWORD': 'mypassword',
+        'USER': 'vmsadmin',
+        'PASSWORD': '0xdeadbeef',
         'HOST': 'localhost',
```

Comment:
syncdb fails with bad user and password, without this change.


On Mon, Oct 19, 2015 at 7:07 AM, Tapasweni Pathak <notifications@github.com>
wrote:

> In vms/vms/settings.py
> <https://github.com/systers/vms/pull/155#discussion_r42362979>:
>
> > @@ -61,8 +61,8 @@
> >      'default': {
> >          'ENGINE': 'django.db.backends.postgresql_psycopg2',
> >          'NAME': 'vms',
> > -        'USER': 'myuser',
> > -        'PASSWORD': 'mypassword',
> > +        'USER': 'vmsadmin',
> > +        'PASSWORD': '0xdeadbeef',
> >          'HOST': 'localhost',
>
> @nkalpana <https://github.com/nkalpana> How does this help?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/systers/vms/pull/155/files#r42362979>.
>